packages/containers/src/GuidedTour/GuidedTour.container.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import omit from 'lodash/omit';
+import { cmfConnect } from '@talend/react-cmf';
+import GuidedTour from '@talend/react-components/lib/GuidedTour';
+
+class GuidedTourContainer extends React.Component {
+	static displayName = 'Container(GuidedTour)';
+
+	static propTypes = {
+		...cmfConnect.propTypes,
+		show: PropTypes.bool,
+		steps: PropTypes.oneOf([PropTypes.array, PropTypes.func]),
+		onClose: PropTypes.func,
+	};
+
+	constructor(props) {
```

Comment:
same here  for the class-transform-properties ;)

Reply:
Ok